### PR TITLE
Governance v2

### PR DESCRIPTION
**Pull request by cryptosunshine**
- The governance page style has been revised

- Add POS mining pool locking function

- Add POS mining pool voting function

- The voting module can vote for an option individually

- Cip 107, add Storage Point Porp option

- - -
This change is [Reviewable](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284)

### CODE REVIEW

File: dapps/governance/src/modules/Dashboard/Pos/LockModal.tsx [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+                        </div>
+                        <div className="mt-[8px] text-[14px] flex justify-between">
+                            <div className="text-[#898D9A]">Est. unlock at:</div>
+                            <div className="text-[#3D3F4C]">{selectIndex >= 0 ? TimeToUnlock[selectIndex]?.unLockTime : '--'}</div>
+                        </div>
+                    </>
+                }
+
+
+                {/* <div className="mt-[16px] flex flex-row justify-between items-center">
```
Comment (angelia-yuqi-personal) [inline dapps/governance/src/modules/Dashboard/Pos/LockModal.tsx:305]: remove the comment code

File: dapps/governance/src/modules/Vote/RewardInterestRate/VotingResult.tsx [hunk 2]
```diff
@@ -63,221 +79,257 @@ const Result: React.FC<{
...
             <Result
-                type="Reward of block"
+                type="PoW block rewards"
                 voteDetail={currentVote?.powBaseReward}
                 preVoteDetail={preVote?.powBaseReward}
+                prepreVoteDetail={prepreVote?.powBaseReward}
                 posStakeForVotes={posStakeForVotes}
-                onClickPreValTip={() => showTipModal(<PowPreviousVotingRewardTipContent />)}
-                onClickVotingValTip={() => showTipModal(<PowVotingRewardTipContent />)}
+            // onClickPreValTip={() => showTipModal(<PowPreviousVotingRewardTipContent />)}
```
Comment (angelia-yuqi-personal) [inline dapps/governance/src/modules/Vote/RewardInterestRate/VotingResult.tsx:208]: remove comment code

File: dapps/governance/src/modules/Vote/RewardInterestRate/VotingResult.tsx [hunk 3]
```diff
@@ -63,221 +79,257 @@ const Result: React.FC<{
...
-const PowPreviousVotingRewardTipContent: React.FC = memo(() => {
-    return (
-        <>
-            <div className="text-[16px] leading-[22px] font-medium text-[#3D3F4C]">Previous voting reward (PoW):</div>
-            <div className="mt-[8px] text-[14px] leading-[21px] text-[#898D9A]">The PoW block rewards of the most recent voting,</div>
-            <div className="text-[14px] leading-[21px] text-[#898D9A]">calculated from the previous round of voting.</div>
-        </>
-    );
-});
+// const PowPreviousVotingRewardTipContent: React.FC = memo(() => {
```
Comment (angelia-yuqi-personal) [inline dapps/governance/src/modules/Vote/RewardInterestRate/VotingResult.tsx:233]: remove comment code

File: dapps/governance/src/store/index.ts [hunk 4]
```diff
@@ -10,7 +10,8 @@ export * from './rewardInterestRate';
 
 export const startTrack = () => {
     const unstakes: Array<() => void> = [];
-    unstakes.push(startTrackBalance(), startTrackPosAccount(), startTrackBlockNumber(), startTrackDaysToUnlock(), startTrackUnlockBlockNumber());
+    // startTrackBalance(), startTrackPosAccount(), startTrackBlockNumber(), startTrackDaysToUnlock(), startTrackUnlockBlockNumber(),
```
Comment (angelia-yuqi-personal) [inline dapps/governance/src/store/index.ts:13]: remove comment code

File: dapps/governance/src/store/proposalList.ts [hunk 5]
```diff
@@ -83,7 +89,7 @@ export const startTrackProposalList = intervalFetchChain(
         ],
     },
     {
-        intervalTime: 30000,
+        intervalTime: 10000,
```
Comment (angelia-yuqi-personal) [inline dapps/governance/src/store/proposalList.ts:92]: why change the interval from 30000 to 10000?

File: dapps/governance/src/store/proposalList.ts [hunk 6]
```diff
@@ -166,7 +269,7 @@ export const startTrackOpenedProposal = () => {
             });
         }
         fetchOpenedProposal();
-        interval = setInterval(fetchOpenedProposal, 2000) as unknown as number;
+        interval = setInterval(fetchOpenedProposal, 5000) as unknown as number;
```
Comment (angelia-yuqi-personal) [inline dapps/governance/src/store/proposalList.ts:272]: why change the interval from 2000 to 5000?

**Review by cryptosunshine — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284)* status: 0 of 40 files reviewed, 7 unresolved discussions (waiting on @angelia-yuqi-personal)

___
*[`dapps/governance/src/modules/Dashboard/Pos/LockModal.tsx` line 151 at r1](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284#gh-1366634904:-NhBDJ5C5mObWFV2_S1y:bhfb539) ([raw file](https://github.com/Conflux-Chain/conflux-dapps/blob/0e54ecf33b21e12e391a2957a39e834e926f1864/dapps/governance/src/modules/Dashboard/Pos/LockModal.tsx#L151)):*
<details><summary><i>Previously, angelia-yuqi-personal (Angelia) wrote…</i></summary><blockquote>

What does ‘more’ mean?
</blockquote></details>

redundant code
___
*[`dapps/governance/src/modules/Dashboard/Pos/LockModal.tsx` line 305 at r1](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284#gh-1366637803:-NhBGQcg7ciNSwgqNs_A:b32wtd) ([raw file](https://github.com/Conflux-Chain/conflux-dapps/blob/0e54ecf33b21e12e391a2957a39e834e926f1864/dapps/governance/src/modules/Dashboard/Pos/LockModal.tsx#L305)):*
<details><summary><i>Previously, angelia-yuqi-personal (Angelia) wrote…</i></summary><blockquote>

remove the comment code
</blockquote></details>

ok
___
*[`dapps/governance/src/modules/Vote/RewardInterestRate/VotingResult.tsx` line 208 at r1](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284#gh-1366647861:-NhBGXNZDJZyRO90tdL7:b32wtd) ([raw file](https://github.com/Conflux-Chain/conflux-dapps/blob/0e54ecf33b21e12e391a2957a39e834e926f1864/dapps/governance/src/modules/Vote/RewardInterestRate/VotingResult.tsx#L208)):*
<details><summary><i>Previously, angelia-yuqi-personal (Angelia) wrote…</i></summary><blockquote>

remove comment code
</blockquote></details>

ok
___
*[`dapps/governance/src/modules/Vote/RewardInterestRate/VotingResult.tsx` line 233 at r1](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284#gh-1366648301:-NhBGYV118E_Jqyvz_zD:b32wtd) ([raw file](https://github.com/Conflux-Chain/conflux-dapps/blob/0e54ecf33b21e12e391a2957a39e834e926f1864/dapps/governance/src/modules/Vote/RewardInterestRate/VotingResult.tsx#L233)):*
<details><summary><i>Previously, angelia-yuqi-personal (Angelia) wrote…</i></summary><blockquote>

remove comment code
</blockquote></details>

ok
___
*[`dapps/governance/src/store/index.ts` line 13 at r1](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284#gh-1366650099:-NhBG_dHA1p76UB1sEtz:b32wtd) ([raw file](https://github.com/Conflux-Chain/conflux-dapps/blob/0e54ecf33b21e12e391a2957a39e834e926f1864/dapps/governance/src/store/index.ts#L13)):*
<details><summary><i>Previously, angelia-yuqi-personal (Angelia) wrote…</i></summary><blockquote>

remove comment code
</blockquote></details>

ok
___
*[`dapps/governance/src/store/proposalList.ts` line 92 at r1](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284#gh-1366651820:-NhBEhP-8FPTljyX6UTc:bswab7o) ([raw file](https://github.com/Conflux-Chain/conflux-dapps/blob/0e54ecf33b21e12e391a2957a39e834e926f1864/dapps/governance/src/store/proposalList.ts#L92)):*
<details><summary><i>Previously, angelia-yuqi-personal (Angelia) wrote…</i></summary><blockquote>

why change the interval from 30000 to 10000?
</blockquote></details>

Pos voting rights need to be calculated. This was not previously necessary, thus reducing request times.
___
*[`dapps/governance/src/store/proposalList.ts` line 272 at r1](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284#gh-1366652452:-NhBFhw30sPRGbER5MnI:b-fal5zt) ([raw file](https://github.com/Conflux-Chain/conflux-dapps/blob/0e54ecf33b21e12e391a2957a39e834e926f1864/dapps/governance/src/store/proposalList.ts#L272)):*
<details><summary><i>Previously, angelia-yuqi-personal (Angelia) wrote…</i></summary><blockquote>

why change the interval from 2000 to 5000?
</blockquote></details>

Frequent requests, reduce frequency.

**Review by angelia-yuqi-personal — APPROVED** [review]

**Review by angelia-yuqi-personal — COMMENTED** [review]
> Reviewed 36 of 40 files at r1, 4 of 4 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/Conflux-Chain/conflux-dapps/284)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**